checkov/common/output/baseline.py

```diff
@@ -56,9 +58,9 @@ def to_dict(self) -> dict[str, Any]:
             formatted_findings = []
             for finding in findings:
                 formatted_findings.append({"resource": finding["resource"], "check_ids": finding["check_ids"]})
-            failed_checks_list.append({"file": file, "findings": formatted_findings})
+            failed_checks_list.append({"file": file, "findings": sorted(formatted_findings, key=itemgetter("resource"))})
 
-        resp = {"failed_checks": failed_checks_list}
+        resp = {"failed_checks": sorted(failed_checks_list, key=itemgetter("file"))}
```

Comment:
@gruebel Within the same files, will the resources keep their order?

Reply:
yes, I made sure of it, two lines up `sorted(formatted_findings, key=itemgetter("resource"))`